Create config.m4

madz0 opened the pull request:
When compiling for the installed cpp-driver, one might get configure: error: Unable to load libcassandra error. In config.log detailed error says undefined reference to `deflateInit_' so I added -lz to fix this

madz0 committed:
Create config.m4

When compiling for the installed cpp-driver, one might get configure: error: Unable to load libcassandra error. In config.log detailed error says undefined reference to `deflateInit_' so I added -lz to fix this

mpenick commented on ext/config.m4:
```diff
@@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
     done
   fi
 
+#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug
```
Thanks for the pull request. Please move this comment into the commit message.

madz0 replied:
Thanks for the regards. Would you like me to amend df8d7b2 commit or just create another commit with the mentioned comment? I just created a comment on the commit.

mpenick replied:
Amend would be preferred, but either way is fine.

madz0 replied:
My previous commit was changed due to some auto generated build files. I
committed a new one with the provided commit message. Hope this does the
job here is the commit id 842915e04e521d9e62edda7187d7098ffdcefdf7
and this is the direct link
https://github.com/mohamad-z/php-driver/commit/842915e04e521d9e62edda7187d7098ffdcefdf7

On Mon, Dec 21, 2015 at 6:41 PM, Michael Penick notifications@github.com
wrote:

> In ext/config.m4
> https://github.com/datastax/php-driver/pull/56#discussion_r48154135:
> 
> > @@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
> >      done
> >    fi
> > 
> > +#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug
> 
> Amend would be preferred, but either way is fine.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/datastax/php-driver/pull/56/files#r48154135.

Commit:
> @@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then

Commit:
> @@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
>      done
>    fi
>  
> +#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug

mpenick commented:
Not all builds require "-lz". Could you add this as an optional library to config.m4 (without the comment please)? Otherwise, you can do this via:

``` sh
LDFLAGS='-lz' ./configure
```

mpenick commented:
Let me know if you're going make it an optional library or please close the pull request.

madz0 commented:
Ok, I see. Unfortunately I'm so busy for couple of days. If you prefer then
I close the pull for now, but let me know if you wanted this to be
implemented and next time no comment  ;)

On Fri, May 6, 2016 at 8:16 PM, Michael Penick notifications@github.com
wrote:

> Let me know if you're going make it an optional library or please close
> the pull request.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/datastax/php-driver/pull/56#issuecomment-217480568
